docs/safety.md

```diff
@@ -275,7 +275,7 @@ Other ways of obtaining SCBs for a node's channels are
 
 Because the backup file is encrypted with a key from the seed the node was
 created with, it can safely be stored on a cloud storage or any other storage
-medium. Many consumer focused wallet smartphone apps automatically store a
+medium. Many consumers focused wallet smartphone apps automatically store a
```

Comment:
this was correct as it was "consumer focused wallet smartphone apps"

Reply:
Good point. I corrected it.